crates/driver/src/infra/api/routes/quote/dto/quote.rs

```diff
@@ -23,6 +36,11 @@ impl Quote {
             solver: quote.solver.0,
             gas: quote.gas.map(|gas| gas.0.as_u64()),
             tx_origin: quote.tx_origin.map(|addr| addr.0),
+            jit_orders: quote
+                .jit_orders
+                .into_iter()
+                .map(|order| order.into())
+                .collect(),
```

Comment:
nit: `jit_orders: quote.jit_orders.into_iter().map(Into::into).collect()` if you like one-liners